src/schema/v2/me/createUserInterestMutation.ts

```diff
@@ -68,9 +85,14 @@ export const createUserInterestMutation = mutationWithClientMutationId<
     )
 
     try {
-      const userInterest: UserInterest = await createUserInterestLoader?.(
-        userInterestInput
-      )
+      let userInterest: UserInterest
+      if (args.userId) {
+        // create interest for another user
+        userInterest = await createUserInterestLoader?.(userInterestInput)
```

Comment:
Did we consider breaking this out into its own mutation? I would expect we would want to keep mutations and queries under `/schema/v2/me` limited to `v1/me/` related endpoints in `gravity` and user-initiated operations.

Reply:
@mzikherman and I briefly spoke about the options (initially in this [thread](https://artsy.slack.com/archives/CA8SANW3W/p1669672810264529)) and then a bit more after standup.

I was initially looking to create new mutations for this but then during our conversation, Matt made a point that this is somewhat of a preference decision between; adding slightly more complexity to the existing mutations but keeping the schema leaner `vs` having another set of mutations and expanding the schema. Keeping the schema leaner seemed like the better and cleaner option so I went with that.

Reply:
Yea, the alternative would be to keep this mutation as-is (and it can't be renamed b/c that breaks existing clients), and create _new_ ones that are called `createUserInterestForUser` (or something). It felt nice to be able to 'slightly complicate' an existing Metaphysics resolver in order to accommodate the different modes we want and not create new mutations.

But it's just preference! It feels nice to me though, all our current mutations under `me` _don't_ have that in their name, so generalizing them is tough w/o doing it this way (and the alternative of creating duplicitous mutations also is eh).

Reply:
See I'm tagged on that thread @ovasdi, so I owe you an apology for the delayed reply here. Proceed in the way that you feel is best here, but I have two strong opinions here: 

Stronger opinion #1: 
- Maybe outside the scope of this PR, but I would argue we should converge on one pattern in situations like this. Splitting implementations is something that should really be mostly boilerplate, like a schema, will ultimately cause unnecessary confusion among developers who will need to try and understand and reason about why and when one implementation is used over the other. 

Less strong opinion #2:
- We've already somewhat embraced splitting the schema based on admin operations vs user driver, `me` operations:
 - [update me mutation](https://github.com/artsy/metaphysics/blob/a33edaa0a2bf53c110fdfc032cf19c9fcaf2e57b/src/schema/v2/me/update_me_mutation.ts)
 - [admin update mutation](https://github.com/artsy/metaphysics/blob/a33edaa0a2bf53c110fdfc032cf19c9fcaf2e57b/src/schema/v2/users/updateUserMutation.ts)

I would argue this will make aditional development easier as I have less backwards compatibility to worry about when making admin related changes as the effect would likely be limited to internal clients.